dist/src/main/resources/api/cluster-api.yaml

```diff
@@ -0,0 +1,371 @@
+openapi: "3.0.2"
+info:
+  title: Cluster Topology Management API
+  version: "1.0"
+  description: API for managing cluster membership and partition distribution.
+servers:
+  - url: "{schema}://{host}:{port}/actuator/cluster"
+    variables:
+      host:
+        default: localhost
+        description: Management server hostname
+      port:
+        default: "9600"
+        description: Management server port
+      schema:
+        default: http
+        description: Management server schema
+paths:
+  /brokers/{brokerId}:
+    post:
+      summary: Add a broker to the cluster
+      description: Add a broker with the given brokerId to the cluster. The broker must be running
+                   to complete the operation.
```

Comment:
:question: Since we now have one request `POST /brokers` that adds/removes brokers and redistribute partitions, I'm wondering if we need separate request to add/remove individual broker.

Reply:
I'd consider this a nice-to-have. It's certainly not needed but could be handy if a controller just knows that they need to remove a specific broker and nothing else.

Reply:
:+1: It would be useful if we add more fine grained operations to change the topology.